docs/sources/vendor/Veeam/veeam.md

```diff
@@ -0,0 +1,24 @@
+# Veeam
+
+## Key facts
+
+* MSG Format based filter
+* rfc5424 default port 514
+
+## Links
+
+| Ref            | Link                                                                                                    |
+|----------------|---------------------------------------------------------------------------------------------------------|
+| Splunk Add-on | <https://splunkbase.splunk.com/app/7312/>                                                              |
```

Comment:
it's developer supported add-on probably we don't need to mention it on documentation for SC4S. @rjha-splunk @mstopa-splunk

Reply:
We will remove this section